PowerUp/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion '23.0.1'
+    buildToolsVersion '26.0.2'
```

Comment:
is there a reason you updated the buildToolsVersion?

Reply:
`gradle:3.0.1` requires minimum 26 or 25. Updated it to version that android studio recommended

Reply:
@tanvidadu Android studio 3.0+ uses gradle version 4.1 or higher. Also , plugin version as 3.0.1 along with buildtoolversion to 26.0.2 or higher. It's good to update the project to latest improvements.